scripts/merge-and-build.sh

```diff
@@ -84,17 +84,42 @@ git clone git@github.com:openshift/ose.git
 cd ose
 if [ "${OSE_VERSION}" == "${OSE_MASTER}" ] ; then
 ## Remove git checkout once master is 3.6
-  git checkout -q enterprise-${OSE_VERSION}
+  git checkout -q fake-master
   git remote add upstream git@github.com:openshift/origin.git --no-tags
   git fetch --all
 
+  # TODO: Once we hard-reset master to contain Origin commits, the carry commits, and the latest tito tag commit
+  # all previous tags are going to be dropped. During the migration, I think we will need to manually tag the
+  # last tito commit or maybe not.
+  last_tag="$( git describe --abbrev=0 --tags )"
+  regex="Automatic commit of package [atomic-openshift] release"
+  last_commit_subject="$( git log HEAD~1..HEAD --pretty=%s )"
+  if [[ ! "${last_commit_subject}" =~ "${regex}" ]]; then
```

Comment:
nit: technically `${regex}` is a literal string and not a regex here
bug: don't we need a `*` at the end?

```sh
if [[ ! "${last_commit_subject}" =~ "${prefix}"* ]]; then
```